src/CommonDropdown.php

```diff
@@ -60,7 +60,8 @@ abstract class CommonDropdown extends CommonDBTM
     /**
      * @since 0.85
      *
-     * @param $nb
+     * @param int $nb
+     * @return string
      **/
```

Comment:
This doc block is probably identical to the parent doc block. It can be removed.